### PR TITLE
Update gltf-pipeline to 3.0.2

### PR DESCRIPTION
**Pull request by ebogo1**
Updates gltf-pipeline to the latest version.

Should this target `master` instead? I believe there are two failing specs (embedded binary KTX/CRN textures) related to https://github.com/CesiumGS/gltf-pipeline/pull/550 so I targeted the new `ktx2-staging` branch instead.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ebogo1!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Changes to third party files were made.
    * Looks like a file in one of our ThirdParty folders (`ThirdParty/, Source/ThirdParty/`) has been added or modified. Please verify that it has a section in [LICENSE.md](https://github.com/CesiumGS/cesium/blob/master/LICENSE.md) and that its license information is up to date with this new version.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+### 1.81 - 2021-05-01
+
+##### Additions :tada:
+
+- Updated [gltf-pipeline](https://github.com/CesiumGS/gltf-pipeline) to 3.0.2. [#9477](https://github.com/CesiumGS/cesium/pull/9477)
```
Comment (lilleyse) [inline CHANGES.md:7]: No need to include this change in `CHANGES.md`. There aren't any user facing changes as part of this PR.

**Comment by lilleyse** [comment]
> Should this target `master` instead?

I think it needs to target the staging branch otherwise KTX1 and CRN in glTF will break prior to us formally removing them (as you noted by the unit tests failing)

**Comment by lilleyse** [comment]
@ebogo1 it's also fine to delete the failing tests. At some point they'll need to be deleted, might as well be now so tests pass locally.

File: CHANGES.md (hunk 1 above)
Comment (ebogo1) [reply CHANGES.md:7]: Just to double check, I was basing this off of https://github.com/CesiumGS/cesium/pull/6805/files. Should I still remove the CHANGES.md changes here?

Comment (lilleyse) [reply CHANGES.md:7]: Ah, that older change was probably unnecessary as well. You should still remove the CHANGES.md entry here.

**Comment by ebogo1** [comment]
@lilleyse Should be ready once 893eadd; I removed the two failing specs and undid the CHANGES.md section.

**Pull request merged**